test/functional/pdo_sqlsrv/PDO38_FetchBound.phpt

```diff
@@ -99,50 +91,50 @@ try {
 
         var_dump($stmt2->execute());
         if ($idx == 0) {   // bindColumn()s after execute() has been called at least once
-            $stmt2->bindColumn('txt', $col1);
+            $stmt2->bindColumn('txt', $col);
         }
         var_dump($stmt2->fetch(PDO::FETCH_BOUND));
         $stmt2->closeCursor();
 
         var_dump($stmt3->execute());
         if ($idx == 0) {   // bindColumn()s after execute() has been called at least once
-            $stmt3->bindColumn('idx', $col2);
+            $stmt3->bindColumn('idx', $idxCol);
         }
         var_dump($stmt3->fetch(PDO::FETCH_BOUND));
         $stmt3->closeCursor();
 
-        var_dump(array($col2=>$col1));
+        var_dump(array($idxCol=>$col));
     }
 
     echo "===REBIND/SAME===\n";
 
-    $stmt3->bindColumn('idx', $col1);
+    $stmt3->bindColumn('idx', $col);
     foreach ($data as $idx => $txt) {
         var_dump(array($idx=>$txt));
         var_dump($stmt2->execute());
         var_dump($stmt2->fetch(PDO::FETCH_BOUND));
         $stmt2->closeCursor();
 
-        var_dump($col1);
+        var_dump($col);
         var_dump($stmt3->execute());
         var_dump($stmt3->fetch(PDO::FETCH_BOUND));
         $stmt3->closeCursor();
-        var_dump($col1);
+        var_dump($col);
     }
 
     echo "===REBIND/CONFLICT===\n";
 
-    $stmt1->bindColumn('idx', $col1);
-    $stmt1->bindColumn('txt', $col1);
+    $stmt1->bindColumn('idx', $col);
+    $stmt1->bindColumn('txt', $col);
     $stmt1->execute();
-    $col1s = array();
+    $cols = array();
     while ($stmt1->fetch(PDO::FETCH_BOUND)) {
-        array_push($col1s, $col1);
+        array_push($cols, $col);
     }
     if (isColEncrypted()) {
-        sort($col1s);
+        sort($cols);
```

Comment:
why sorting required?

Reply:
I think I just didn't see this one when I was refactoring. I'll fix it.